src/api/main.py

```diff
@@ -139,8 +139,12 @@ def run_rwd_nhd(lat, lon):
         input_shp_path = os.path.join(output_path, 'New_Outlet.shp')
 
         output = {
-            'watershed': load_json(wshed_shp_path, output_path, simplify),
-            'input_pt': load_json(input_shp_path, output_path)
+            'watershed': load_json(wshed_shp_path, output_path, simplify,
+                                   # From NAD83 / Conus Albers to WGS 84 Latlong
+                                   from_epsg=4269, to_epsg=4326),
```

Comment:
The epsg code which the source data is reprojected from is a different (5070).  Did this new code come from an actual shapeful that was produced from nhd rwd?

https://github.com/WikiWatershed/rapid-watershed-delineation/pull/37/commits/065ee7b309a42161694dd97a33cacac80dcbe666#diff-0179452433a9687f71f6d0aa8e94bcadR116